pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileIngestService.java

```diff
@@ -137,8 +137,8 @@ long getMinTimeInMillisToUpdateFile() {
         return Long.parseLong(getProperty(MIN_TIME_IN_MILLIS_TO_UPDATE_FILE_KEY, "5000"));
     }
 
-    long getDeleteCompletedFilesIntervalInMinutes() {
-        return Long.parseLong(getProperty(DELETE_COMPLETED_FILES_INTERVAL_IN_MINUTES_KEY, "720"));
+    long getDeleteCompletedFilesIntervalInSeconds() {
+        return Long.parseLong(getProperty(DELETE_COMPLETED_FILES_INTERVAL_IN_SECONDS_KEY, "43200"));
```

Comment:
is it like are we trying to delete completed file every 12 hours? @dada-dell-emc

Reply:
Yes